app/src/androidTest/java/net/skyscanner/backpack/docs/GenerateScreenshots.kt

```diff
@@ -45,40 +47,37 @@ open class GenerateScreenshots(
     }
 
     @get:Rule
-    var activityRule = ActivityTestRule(BpkBaseActivity::class.java, true, false)
-
-    @get:Rule
-    val composeTestRule = AndroidComposeTestRule(activityRule) { it.activity }
+    val composeTestRule = createAndroidComposeRule<AppCompatActivity>()
 
     @Test
     fun testTakeScreenshot() {
-        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
-        runActivityAndTakeScreenshot()
+        composeTestRule.runOnUiThread { AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO) }
```

Comment:
It works only from UI thread, and AndroidComposeRule is not there by default. `runOnUiThread` is blocking.